Mathlib/GroupTheory/SemidirectProduct.lean

```diff
@@ -259,4 +282,29 @@ theorem map_comp_inr : (map f₁ f₂ h).comp inr = inr.comp f₂ := by ext <;>
 
 end Map
 
+section Congr
+
+variable {N₁ : Type*} {G₁ : Type*} [Group N₁] [Group G₁] {φ₁ : G₁ →* MulAut N₁}
+
+/-- Define an isomorphism from `N ⋊[φ] G` to `N₁ ⋊[φ₁] G₁` given isomorphisms `N ≃* N₁` and
+  `G ≃* G₁` that satisfy a commutativity condition `∀ n g, f₁ (φ g n) = φ₁ (f₂ g) (f₁ n)`. -/
+@[simps]
+def congr (f₁ : N ≃* N₁) (f₂ : G ≃* G₁) (h : ∀ g : G, (φ g).trans f₁ = f₁.trans (φ₁ (f₂ g))) :
+    N ⋊[φ] G ≃* N₁ ⋊[φ₁] G₁ where
```

Comment:
Could you please rename the variables so that the role of subscripts is more symmetrical. I would imagine that both domain and codomains have subscripts, but then the equivs should not have (numbered subscripts).

Reply:
Yeah, I was just following the existing numbering, but you're right that it should be fixed.